miden-tx/src/tests/kernel_tests/test_faucet.rs

```diff
@@ -453,15 +428,16 @@ fn test_burn_fungible_asset_insufficient_input_amount() {
 #[ignore]
 #[test]
 fn test_burn_non_fungible_asset_succeeds() {
-    let (tx_inputs, tx_args) = mock_inputs(
-        MockAccountType::NonFungibleFaucet {
-            acct_id: ACCOUNT_ID_NON_FUNGIBLE_FAUCET_ON_CHAIN,
-            nonce: ONE,
-            empty_reserved_slot: false,
-        },
-        AssetPreservationStatus::TooManyNonFungibleInput,
-    );
-    let non_fungible_asset_burnt = non_fungible_asset_2(ACCOUNT_ID_NON_FUNGIBLE_FAUCET_ON_CHAIN);
+    let acc_type = MockAccountType::NonFungibleFaucet {
+        acct_id: ACCOUNT_ID_NON_FUNGIBLE_FAUCET_ON_CHAIN,
+        nonce: ONE,
+        empty_reserved_slot: false,
+    };
+
+    let tx_context = TransactionContextBuilder::with_acc_type(acc_type).build();
```

Comment:
should we use `with_mock_notes` and `AssetPreservationStatus::TooManyNonFungibleInput`

Reply:
In this test notes are not involved so mocking them is unnecessary.